spec/lib/embed/viewer/media_spec.rb

```diff
@@ -17,6 +19,7 @@
   end
 
   describe '#body_html' do
+    before { stub_purl_response_with_fixture(video_purl) }
     it 'includes some dummy content' do
```

Comment:
would it be more appropriate to test for specific contents now that the fixture objects have them and the code exists to parse it?